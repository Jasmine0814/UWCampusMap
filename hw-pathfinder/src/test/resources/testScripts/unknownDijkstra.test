# test unknown start or dest

CreateGraph Graph1

AddNode Graph1 A
AddNode Graph1 B
AddNode Graph1 C

AddEdge Graph1 A B 1.2344

FindPath Graph1 A B

FindPath Graph1 B D

FindPath Graph1 D B

FindPath Graph1 D D