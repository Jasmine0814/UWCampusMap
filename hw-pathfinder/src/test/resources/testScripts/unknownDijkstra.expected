# test unknown start or dest

created graph Graph1

added node A to Graph1
added node B to Graph1
added node C to Graph1

added edge 1.234 from A to B in Graph1

path from A to B:
A to B with weight 1.234
total cost: 1.234

unknown: D

unknown: D

unknown: D
unknown: D